{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/a/SMTB18_Project/Data/Heteromers\n"
     ]
    }
   ],
   "source": [
    "# Move to Homomers folder\n",
    "%cd ../Heteromers"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Bash pipeline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "https://files.rcsb.org/download/5cy3.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/3peu.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/1zhi.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/1A3N.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/1eer.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/1EFV.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/2E0X.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/3a7a.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/3Zhe.pdb\n",
      "-1 / unknownhttps://files.rcsb.org/download/3NVO.pdb\n",
      "-1 / unknownP29295\n",
      "https://www.uniprot.org/uniprot/P29295.fasta\n",
      "100% [..............................................................] 642 / 642P20449\n",
      "https://www.uniprot.org/uniprot/P20449.fasta\n",
      "100% [..............................................................] 630 / 630P54784\n",
      "https://www.uniprot.org/uniprot/P54784.fasta\n",
      "100% [............................................................] 1074 / 1074P68871\n",
      "https://www.uniprot.org/uniprot/P68871.fasta\n",
      "100% [..............................................................] 236 / 236P01588\n",
      "https://www.uniprot.org/uniprot/P01588.fasta\n",
      "100% [..............................................................] 274 / 274P13804\n",
      "https://www.uniprot.org/uniprot/P13804.fasta\n",
      "100% [..............................................................] 463 / 463P18956\n",
      "https://www.uniprot.org/uniprot/P18956.fasta\n",
      "100% [..............................................................] 702 / 702PoA6T9\n",
      "https://www.uniprot.org/uniprot/PoA6T9.fasta\n",
      "100% [..............................................................] 248 / 248G5ECF1\n",
      "https://www.uniprot.org/uniprot/G5ECF1.fasta\n",
      "100% [..............................................................] 673 / 673Q9XV58\n",
      "https://www.uniprot.org/uniprot/Q9XV58.fasta\n",
      "100% [..............................................................] 368 / 368"
     ]
    }
   ],
   "source": [
    "# Retreive protein sequence\n",
    "!python3 ../../Scripts/01.getSequences.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n"
     ]
    }
   ],
   "source": [
    "# Write the blast code\n",
    "!python3 ../../Scripts/02.getBLAST.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n",
      "Warning: [blastp] Unavailable feature GNUTLS\n",
      "Critical: [blastp] Unavailable feature GNUTLS\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#2000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#3000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (302.131) [blastp] SSOCK#4000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#5000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#6000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#7000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#8000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#9000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (302.131) [blastp] SSOCK#10000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#11000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#12000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: (302.131) [blastp] SSOCK#13000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#14000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (302.131) [blastp] SSOCK#15000[?]: [SOCK::Connect]  Failed to initialize secure session\n",
      "Error: (303.7) [blastp] [URL_Connect; https://www.ncbi.nlm.nih.gov:443/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Failed to connect: Not supported\n",
      "Error: (307.1) [blastp] [HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu]  Too many failed attempts (3), giving up\n",
      "Warning: (301.21) [blastp] [CONN_Flush(HTTP; https://www.ncbi.nlm.nih.gov/Service/dispd.cgi?service=blast4&address=wbadmin-OptiPlex-790(127.0.1.1)&platform=x86_64-unknown-linux-gnu)]  Failed to flush: Not supported\n",
      "Error: (308.5) [blastp] [blast4]  Service not found\n",
      "Error: (315.2) [blastp] CConn_Streambuf::CConn_Streambuf():  NULL connector: Unknown\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 0: stack is empty: cannot close output stream\n",
      "Error: (CIOException::eFlush) [blastp] COStreamBuffer::Flush: failed\n",
      "Error: (CSerialException::eIoError) [blastp] byte 0: cannot close output stream\n",
      "Error: (802.4) [blastp] Cannot close serializing output stream (CSerialException::eIoError) byte 0: cannot close output stream\n",
      "Error: (802.5) [blastp] CObjectOStream: error at byte 118: stack is empty: write fault\n",
      "Error: NCBI C++ Exception:\n",
      "    T0 \"/opt/conda/conda-bld/blast_1502313036753/work/ncbi-blast-2.6.0+-src/c++/src/serial/rpcbase.cpp\", line 244: Error: ncbi::CRPCClient_Base::x_Ask() - Failed to receive reply after 3 tries\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Run blast code\n",
    "!bash blast_protList.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "B3GWA1 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../B3GWA1 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "B3GWA1\n",
      "mkdir: cannot create directory ‘B3GWA1’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownO75874 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../O75874 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "O75874\n",
      "mkdir: cannot create directory ‘O75874’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownP02766 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../P02766 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "P02766\n",
      "mkdir: cannot create directory ‘P02766’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownP0A6C8 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../P0A6C8 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "P0A6C8\n",
      "mkdir: cannot create directory ‘P0A6C8’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownP32325 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../P32325 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "P32325\n",
      "mkdir: cannot create directory ‘P32325’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownTraceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 18, in <module>\n",
      "    handle = Entrez.efetch(db=\"protein\", id=code,rettype=\"fasta\", retmode=\"text\")\n",
      "  File \"/home/a/.local/lib/python3.5/site-packages/Bio/Entrez/__init__.py\", line 184, in efetch\n",
      "    return _open(cgi, variables, post=post)\n",
      "  File \"/home/a/.local/lib/python3.5/site-packages/Bio/Entrez/__init__.py\", line 533, in _open\n",
      "    raise exception\n",
      "  File \"/home/a/.local/lib/python3.5/site-packages/Bio/Entrez/__init__.py\", line 531, in _open\n",
      "    handle = _urlopen(cgi)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 163, in urlopen\n",
      "    return opener.open(url, data, timeout)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 472, in open\n",
      "    response = meth(req, response)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 582, in http_response\n",
      "    'http', request, response, code, msg, hdrs)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 510, in error\n",
      "    return self._call_chain(*args)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 444, in _call_chain\n",
      "    result = func(*args)\n",
      "  File \"/usr/lib/python3.5/urllib/request.py\", line 590, in http_error_default\n",
      "    raise HTTPError(req.full_url, code, msg, hdrs, fp)\n",
      "urllib.error.HTTPError: HTTP Error 502: Bad Gateway\n",
      "P57743 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../P57743 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "P57743\n",
      "mkdir: cannot create directory ‘P57743’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownPOAFX9 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../POAFX9 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "POAFX9\n",
      "mkdir: cannot create directory ‘POAFX9’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownQ06592 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../Q06592 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "Q06592\n",
      "mkdir: cannot create directory ‘Q06592’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "-1 / unknownQ19905 (1)\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/a/SMTB18_Project/Scripts/03.getHomologous.py\", line 9, in <module>\n",
      "    with open(inputFile, 'r') as f:\n",
      "FileNotFoundError: [Errno 2] No such file or directory: '../Q19905 (1).blast'\n",
      "cat: '*.fasta': No such file or directory\n",
      "rm: cannot remove '*.fasta': No such file or directory\n",
      "Q19905\n",
      "mkdir: cannot create directory ‘Q19905’: File exists\n",
      "/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88\n",
      "  return f(*args, **kwds)\n"
     ]
    }
   ],
   "source": [
    "# Retrive \n",
    "!bash ../../Scripts/concatenate.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Clean alignment\n",
    "%%python3 ../../Scripts/04.cleanAligment.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get Shannons Entropy\n",
    "%%python3 ../../Scripts/05.getShannons.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Fucking KA/KC\n",
    "%%python3 ../../Scripts/06.getRate.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get surface core\n",
    "%%python3 ../../Scripts/07.getSurfaceCore.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get interfaces \n",
    "%%python3 ../../Scripts/08.getInterfaces.py protList.tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Analysis of results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
